### PR TITLE
feat: Employee class

### PR DESCRIPTION
**Pull request by RogerMo01**

### CODE REVIEW

**Comment by RogerMo01** [comment]
What should I implement in al method called ToString() for this class??

**Review by LongJohnSilver1504 — CHANGES_REQUESTED** [review]
Good work, the toString method is for do 
Employee.toString() // and get something like   id-5 Roger Moreno with salary 1mn

File: OOP-Exercises/Employee.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+namespace OOP_Exercises
+{
+    class Employee
+    {
+        public int Id { get; private set; }
+        public string FirstName { get; private set; }
+        public string LastName { get; private set; }
+        public int Salary = 0;
+        public new string ToString => "Id-" + Id + " FirstName-" + FirstName + " LastName-" + LastName + " CurrentSalary-" + Salary;
```
Comment (LongJohnSilver1504) [inline OOP-Exercises/Employee.cs:15]: Why not use string interpolation here

Comment (RogerMo01) [reply OOP-Exercises/Employee.cs:15]: "Void" reason
Cleared Up!

**Pull request merged**